cdap-ui/cypress/fixtures/fll_airport_pipeline2.json

```diff
@@ -35,7 +35,7 @@
                     "label": "Airport_source",
                     "artifact": {
                         "name": "core-plugins",
-                        "version": "2.3.0-SNAPSHOT",
+                        "version": "[2.3.0-SNAPSHOT, 3.0.0-SNAPSHOT)",
```

Comment:
should they both be square brackets or parentheses?

Reply:
nope, this is correct. It's inclusive vs exclusive. Square means inclusive, parentheses means exclusive